docs/devguide/xtypes.rst

```diff
@@ -313,9 +316,9 @@ Final Extensibility
 ..
     Sect<16.3.5>
 
-The third kind of extensibility is @final.
-Annotating a type with @final means that it will not be compatible with (assignable to/from) a type that's structurally different.
-The @final annotation can be used to define types for pre-XTypes compatibility or in situations where the overhead of @mutable or @appendable is unacceptable.
+The third kind of extensibility is final.
+Annotating a type with ``@final`` means that it will not be compatible with (assignable to/from) a type that's structurally different.
```

Comment:
```suggestion
Annotating a type with ``@final`` means that it will not be compatible with (assignable to/from) a type that is structurally different.
```